src/inspector_agent.cc

```diff
@@ -370,6 +370,16 @@ bool IsFilePath(const std::string& path) {
 }
 #endif  // __POSIX__
 
+void ThrowUninitializedInspectorError(Environment* env) {
+  HandleScope scope(env->isolate());
+
+  const char* msg = "This Environment was initialized without a V8::Inspector";
```

Comment:
It would be nicer if this error is defined in `node_errors.h` and documented in errors.md, though I understand that this would not normally show up when you use the Node.js official build so it's more like something that belongs to the doc of the embedder..